lib/utils/initFile.js

```diff
@@ -5,5 +5,5 @@ module.exports = function (file) {
 
     file.s3 = {};
     file.s3.headers = {};
-    file.s3.path = file.path.replace(file.base, "");
+    file.s3.path = file.path.replace(file.base, "").replace(/\\/g, '/');
```

Comment:
I considered that, but I thought it kinda actually hurt the readability. Your call though; I'll push an amended commit. :)